generator/templates/server/server.gotmpl

```diff
@@ -562,3 +596,37 @@ func (s *Server) TLSListener() (net.Listener, error) {
 	}
 	return s.httpsServerL, nil
 }
+
+func (s *Server) interruptChan() chan os.Signal {
```

Comment:
why is this lazily initialized and not wihen everything else in intialized?
initializing it early removes the need for the mutex

Reply:
Ok agreed.